src/main/java/ru/practicum/shareit/booking/BookingServiceImpl.java

```diff
@@ -0,0 +1,145 @@
+package ru.practicum.shareit.booking;
+
+import lombok.RequiredArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.booking.dto.BookingDto;
+import ru.practicum.shareit.booking.mapper.BookingMapper;
+import ru.practicum.shareit.booking.repository.BookingRepository;
+import ru.practicum.shareit.exceptions.MissingException;
+import ru.practicum.shareit.exceptions.ValidationException;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.item.repository.ItemRepository;
+import ru.practicum.shareit.user.model.User;
+import ru.practicum.shareit.user.repository.UserRepository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class BookingServiceImpl implements BookingService {
+
+
+    private final ItemRepository itemRepository;
+    private final UserRepository userRepository;
+    private final BookingRepository bookingRepository;
+    private final BookingMapper bookingMapper;
+
+    @Override
+    public Booking createBooking(BookingDto bookingDto, Integer userId) {
+        Booking booking = bookingMapper.toBooking(bookingDto);
+        Item bookingsItem = itemRepository.findById(bookingDto.getItemId()).orElseThrow(() -> {
+            throw new MissingException("is not exist");
+        });
```

Comment:
Нужно удалить лишние скобки должно быть:
```suggestion
        .orElseThrow(() -> new MissingException("is not exist"));
```